src/funding.ts

```diff
@@ -69,12 +69,17 @@ export const fundAccount = async ({
   account,
   sourceChains,
   sourceTokens,
+  environment,
 }: {
   account: Address;
   sourceChains: string[];
   sourceTokens: string[];
+  environment?: string;
 }) => {
-  if (process.env.LOCAL_TESTNET) {
+  // Only fund accounts when using local testnet or when LOCAL_TESTNET is explicitly set to true
+  const shouldFund = process.env.LOCAL_TESTNET === 'true' || environment === 'local';
```

Comment:
so the reason we have a separate flag for local testnets is that you can use the local orchestrator with live networks which can save the hassle of setting up the full e2e env. imo we should just stick to `LOCAL_TESTNET` for now and if we only use local testnets in the future we can get rid of it and only use `env == 'local'`